dask_cuda/tests/test_local_cuda_cluster.py

```diff
@@ -245,3 +245,15 @@ def get_visible_devices():
             del os.environ["CUDA_VISIBLE_DEVICES"]
         if init_nvmlstatus:
             os.environ["DASK_DISTRIBUTED__DIAGNOSTICS__NVML"] = init_nvmlstatus
+
+
+@gen_test(timeout=20)
+async def test_gpu_uuid():
+    async with LocalCUDACluster(
+        CUDA_VISIBLE_DEVICES=get_gpu_uuid_from_index(0),
+        scheduler_port=0,
+        asynchronous=True,
+    ) as cluster:
+        assert len(cluster.workers) == 1
+        async with Client(cluster, asynchronous=True) as client:
+            await client.wait_for_workers(1)
```

Comment:
For completeness, maybe also check that the correct `CUDA_VISIBLE_DEVICES` is set on the worker?

Reply:
Good idea, done in https://github.com/rapidsai/dask-cuda/pull/780/commits/839d03a0f4c6cb1fc6f18649f74912c87dc79690 .